### PR TITLE
Create VMFS datastore with datastore.create

### PR DESCRIPTION
**Pull request by pietern**
Consolidate this command with datastore.nas.create into the new command
datastore.create. The new name provides symmetry with datastore.remove
to remove a datastore.

### CODE REVIEW

File: govc/datastore/create.go [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+		var option *types.VmfsDatastoreOption
+		for _, e := range options {
+			if _, ok := e.Info.(*types.VmfsDatastoreAllExtentOption); ok {
+				option = &e
+				break
+			}
+		}
+
+		if option == nil {
+			panic("cannot use entire disk")
```
Comment (dougm) [inline govc/datastore/create.go]: Should this be a return error instead?

**Comment by dougm** [comment]
:+1: Looks great to me!

**Comment by dougm** [comment]
Don't forget to update the ChangeLog(s) too!

**Comment by pietern** [comment]
@dougm Fixed the panic and updated the changelog :)

**Comment by dougm** [comment]
ship it!

**Comment by pietern** [comment]
Changed one line after the ship it to fix up datastore creation.

**Pull request merged**